### ISSUE TITLE
Callbacks

### ISSUE THREAD

**Issue by prithagupta** [kiudee/cs-ranking#58]
- [x] Bug in LRScheduler of callbacks
- [x] Create LRSchedular and EarStopping independent of Keras implementation to avoid issues. 
One way would be to inherit from Callback from Keras.

There is an issue with the current LrScheduler. The problem is that it exponentially decreases the learning rate at each epoch after the epoch_drop.
For learning rate of 0.015 and epoch drop=5 and drop percentage =0.9

Epoch 00001: LearningRateScheduler setting learning rate to 0.014999999664723873.
Epoch 00005: LearningRateScheduler setting learning rate to 0.013499999698251487.
Epoch 00006: LearningRateScheduler setting learning rate to 0.012149999476969242.
Epoch 00007: LearningRateScheduler setting learning rate to 0.010934999864548446.
Epoch 00008: LearningRateScheduler setting learning rate to 0.009841500129550696
Epoch 00009: LearningRateScheduler setting learning rate to 0.00885734986513853.
Epoch 00010: LearningRateScheduler setting learning rate to 0.007174453390762211
Epoch 00011: LearningRateScheduler setting learning rate to 0.005811307118274272
While the output should be:
Epoch 00001: LearningRateScheduler setting learning rate to 0.014999999664723873.
Epoch 00005: LearningRateScheduler setting learning rate to 0.013499999698251487.
Epoch 00010: LearningRateScheduler setting learning rate to 0.012149999728426338.
Epoch 00015: LearningRateScheduler setting learning rate to 0.010934999755583704.

**Comment by prithagupta** [comment]
Creat LRSchedular and EarStopping independent of Kera implementation to avoid issues. One way would be to inherit from Callback from keras.
